packages/core/src/testInteg/appBuilder/sidebar/appBuilderNode.test.ts

```diff
@@ -41,9 +40,9 @@ describe('Application Builder', async () => {
     })
 
     after(async () => {
-        sandbox.restore()
         // Restore original status of walkthroughCompleted status
         await globals.globalState.update('aws.toolkit.lambda.walkthroughCompleted', originalWalkThroughState)
+        sinon.restore()
```

Comment:
You will probably want to restore in `afterEach` instead of `after` since a stub can bleed in to an adjacent test

Reply:
ack